README.md

```diff
@@ -1,122 +1,132 @@
+| OAUTH_APP_ID            | The ID of the github OAuth app                                                                                                          |
+| OAUTH_APP_SECRET        | The secret key for the github OAuth app                                                                                                 |
+
+## Running the App
+
+Once the all dependencies have been installed, start the Flask app in development mode within the Poetry environment by running:
+
+```bash
+$ poetry run flask run
+```
+
+You should see output similar to the following:
+
+```bash
+ * Serving Flask app "app" (lazy loading)
+ * Environment: development
+ * Debug mode: on
+ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+ * Restarting with fsevents reloader
+ * Debugger is active!
+ * Debugger PIN: 226-556-590
+```
+
+Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+### Running the app using ansible
+
+To run the app using ansible copy the `/ansible/controller_files` folder to the controller node
+
+Then update the inventory to include any additional host nodes and run the below command from inside the `controller_files` folder
+
+`ansible-playbook playbook.yml -i inventory`
+
+### Running the app using docker
+
+To run the app using docker ensure docker daemon is running on your machine then
+
+#### Development
+
+Build the container by running `docker-compose -f docker-compose-dev.yml build`
+
+Run the container using `docker-compose -f docker-compose-dev.yml -p 'todo-app-dev' up`
+
+The dev site will then be available on `localhost:5000`
+
+#### Production
+
+Build the container by running `docker-compose -f docker-compose-prod.yml build`
+
+Run the container using `docker-compose -f docker-compose-prod.yml -p 'todo-app-prod' up`
+
+The production site will then be available on `0.0.0.0` (port 80)
+
+## Running the tests
+
+### Unit and Integration Tests
+
+This project uses `pytest` for unit testing, run the tests with the command `poetry run pytest tests` or watch the tests using `poetry run ptw tests`.
+
+To run individual tests append the test dir to the command used.
+
+### E2E tests
+
+To run the end to end tests run the command `poetry run pytest tests_e2e`
+
+### Dockerised Tests
+
+The tests can also be run using docker
+
+For end to end tests run `docker-compose -f docker-compose-test-e2e.yml -p 'todo-app-test-e2e' up`
+
+For unit tests run `docker-compose -f docker-compose-test-unit.yml -p 'todo-app-test-unit' up`
+
+## Deployments
+
+The app has continuous deployment to an azure web app which is triggered on merges to main.
+
+### Deploying to MiniKube
+
+1. Have a running instance of minikube
+2. Create a copy of `kubernetes/secrets-template.yml` at `kubernetes/secrets.yml` and fill in the secret values
+3. Load the built prod container to minikube tagged as `todo-app:prod`

```

Comment:
I think this is missing a reference to the method, perhaps the wording is ambiguous. I understand this as running 
`kubectl run todo-app --image=todo-app:prod`. If that's correct it would still benefit from making it explicit